content/en/dora_metrics/setup/failures.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Understanding Incident Data for DORA Metrics
```

Comment:
This page is about the setup, not understanding the data

Reply:
```suggestion
title: How to Set Up Incident Data for DORA Metrics
```